src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const RENT = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  if (days >= LONG_TERM) {
+    return RENT * days - LONG_TERM_DISCOUNT;
+  } else if (days >= SHORT_TERM) {
```

Comment:
[Checklist/4] ]Prefer `if` with `return` over `if else` to simplify later conditions.